frontend/components/site/sitePublishedBranches.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import publishedBranchActions from "../../actions/publishedBranchActions";
+
+class SitePublishedBranches extends React.Component {
+  componentDidMount() {
+    publishedBranchActions.fetchPublishedBranches({ id: this.props.params.id })
+  }
+
+  publishedBranches() {
+    if (!this.props.publishedBranches) {
+      return []
+    }
+    return this.props.publishedBranches.filter(branch => {
+      return branch.site.id === parseInt(this.props.params.id)
+    })
+  }
+
+  render() {
+    if (this.publishedBranches().length > 0) {
```

Comment:
Do we care about the edge case where we show "loading published branches..." when there are no published branches yet?